Makefile

```diff
@@ -181,8 +181,8 @@ console.log(
 			postinstall: "node ./postinstall.js"
 		},
 		bin: {
-			esy: "bin/esy",
-			esyi: "bin/esyi"
+			esy: "_build/default/esy/bin/esyCommand.exe",
```

Comment:
@andreypopp - are there any concerns with linking directly to the executables? I believe after unpacking, this path is consistent - and linking directly will work on Windows.

Just wondering if there is any reason we're using symlinks that I'm missing or not accounting for here.

Reply:
I think the symlink isn't necessary but there should be a present file at the time npm creates its wrappers for `bin`. We need to fix `Makefile`'s `release` target to create an empty `_build/defaiult/esy/bin/esyCommand.exe` then.